commit 1251c246cf7512367137f7e62d92910af7ba0ab0 (refs/changes/41/5641/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-02-26T18:35:07+00:00 (7 months ago)
    
    TOOLS-2188 bits-upload.sh should pass --compression=none for uncompressed image files
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/deps/eng b/deps/eng
index 7c472bc..cfd827a 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit 7c472bc495ba44ac4cdde0af50ff627021942524
+Subproject commit cfd827a5029c249624efd6fcdee128db2a9d727e
